ui/widgets/controlgroup.js

```diff
@@ -112,6 +112,11 @@ return $.widget( "ui.controlgroup", {
 				.each( function() {
 					var element = $( this );
 					var instance = element[ widget ]( "instance" );
+
+					// If the button is the child of a spinner ignore it
+					if ( widget === "button" && element.parent( ".ui-spinner" ).length ) {
```

Comment:
Seems like this should really be more generic. Something along the lines of if the found widget is inside any other widget which is also inside this controlgroup, then ignore it. The problem is the "which is also inside any other widget" part. I'm not sure there's anything we can do about this right now, but I really don't like that we're writing logic around specific widgets.